contrib/babelfishpg_tsql/sql/sys_functions.sql

```diff
@@ -74,6 +74,42 @@ END;
 $$
 LANGUAGE plpgsql IMMUTABLE;
 
+-- helper functions for XML EXIST(xpath)
+CREATE OR REPLACE FUNCTION sys.bbf_xmlexist(TEXT, ANYELEMENT)
+RETURNS sys.BIT
+AS
+$BODY$
+DECLARE
+    string_arg_datatype text;
+    string_basetype oid;
+    pltsql_quoted_identifier text;
+BEGIN
+    string_arg_datatype := sys.translate_pg_type_to_tsql(pg_typeof($2)::oid);
+    IF string_arg_datatype IS NULL THEN
+        -- for User Defined Datatype, use immediate base type to check for argument datatype validation
```

Comment:
In TSQL creation of User Defined datatype on xml is not allowed, whereas in Babelfish we allow it, now since we allow creating UDT on XML in Babelfish, kept its behaviour same as xml datatype.